src/App.css

```diff
@@ -1,38 +1,156 @@
+
+*, *:before, *:after {box-sizing: inherit}
+
+.sr-only { position: absolute; width: 1px; height: 1px; margin: -1px; border: 0; padding: 0; white-space: nowrap; clip-path: inset(100%); clip: rect(0 0 0 0); overflow: hidden;}
+
+@font-face {
+  font-family: "Hind";
+ 
+  src: local("Hind-Regular"),
+    url("./fonts/Hind-Regular.ttf") format("truetype");
+}
+@font-face {
+  font-family: "Montserrat";
+
+  src: local("Montserrat"),
+    url("./fonts/Montserrat-VariableFont_wght.ttf") format("truetype");
+}
+
+
+html{
+  font-size: 62.5%;
+}
+
+body {
+  background-color: slateblue;
+  font-size: 2rem;
+}
+
+header {
+  text-align: center;
+}
+
+h1{
+  font-size: 6rem;
+  color: white;
+  font-family: Montserrat, sans-serif;
+}
+
+h2{
+  margin: 5px 1px 50px 1px;
+  font-family: Hind, sans-serif;
+  font-weight: 600;
+  font-size: 1.8rem;
+  color: darkslateblue;
+  text-transform: uppercase;
+}
+
+p {
+  font-size: 2rem;
+  font-family: Hind, sans-serif;
   text-align: center;
+  color: white;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+a {
+  text-decoration: none;
+  font-family: Hind, sans-serif;
+  font-size: 1.5rem;
+  font-weight: 600;
+  border: 2px solid darkslateblue;
+  padding: 2.5px 5px;
+  color: darkslateblue;
+  letter-spacing: -0.5px;
+  
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+a:hover{
+  background-color: darkslateblue;
+  color: white;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.link{
+  text-transform: uppercase;
+  position: absolute;
+  bottom: 0;
+  margin-bottom: 5px;
+}
+
+ul{
+  background-color: white;
+  padding: 0;
   display: flex;
-  flex-direction: column;
-  align-items: center;
+  flex-wrap: wrap;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
 }
 
-.App-link {
-  color: #61dafb;
+li{
+  width: calc((100%/3) - 220px);
+  margin: 30px;
+  padding: 25px 2%;
+  min-width: 220px;
+  max-width: 310px;
+}
+
+.itemContainer{
+  position: relative;
+  list-style: none;
+  margin: 20px;
+  padding: 0;
+  box-shadow: 5px 10px 15px grey;
+}
+
+.itemContainer:hover{
+  box-shadow:5px 10px 15px slateblue ;
+}
+
+img {
+  max-width: 100%;
+  opacity: 0.85;
+}
+
+.itemContainer:hover .image{
+  opacity: 1;
+}
+
+.textContainer {
+  margin: 5px 5px 10px 5px;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+form{
+  display: flex;
+  justify-content: center;
+  margin: 50px 0;
+}
+
+input{
+  border: 1px solid black;
+  border-right: none;
+  padding: 5px;
+  border-radius: 5px 0 0 5px;
+  outline: none;
+  width: 50%;
+}
+
+input:focus{
+  color: slateblue;
 }
+
+input::placeholder{
+  color: rgb(99, 156, 212);
+}
+
+.searchBtn{
+  width: 40px;
+  height: 36px;
+  border: 1px solid black;
+  border-radius: 0 5px 5px 0;
+  cursor: pointer;
+}
+
+
```

Comment:
all indenting looks great, but maybe add comments / group CSS by components for extra organization